logtracer/tracing/tracer.py

```diff
@@ -18,6 +17,8 @@
 B3_SAMPLED = 'X-B3-Sampled'
 B3_FLAGS = 'X-B3-Flags'
 GOOGLE_LOAD_BALANCER_TRACE_HEADERS = "X-Cloud-Trace-Context"
+_TRACE_CONTEXT_HEADER_FORMAT = r'([0-9a-f]{32})(\/([0-9a-f]{16}))?(;o=(\d+))?'
```

Comment:
hmm, for me I find this harder to read than if statements, but that is because I don't practice regex too often